test/index.test.js

```diff
@@ -500,18 +500,18 @@ describe('Gets a Bridge Transaction given a bridgeTx: web3TransactionObject and
         assert.lengthOf(transaction.events, 3);
         assert.equal(transaction.events[0].name, "update_collections");
         assert.equal(transaction.events[0].signature, "0x1069152f4f916cbf155ee32a695d92258481944edb5b6fd649718fc1b43e515e");
-        assert.equal(transaction.events[0].arguments.get('sender'), '0xFE90f02331DdF62cb50F5650Dca554b47B37c471');
+        // assert.equal(transaction.events[0].arguments.get('sender'), '0xFE90f02331DdF62cb50F5650Dca554b47B37c471');
```

Comment:
Why this line is commented on? If it is not necessary anymore can it be removed?

Reply:
Good Catch. Thanks Nath